package/MDAnalysis/lib/formats/libdcd.pyx

```diff
@@ -37,6 +38,7 @@ ctypedef int fio_fd;
 ctypedef off_t fio_size_t
 
 ctypedef np.float32_t DTYPE_t
+ctypedef np.float64_t DTYPE_t2
```

Comment:
Please rather name them accordingly `FLOAT_T`, 'DOUBLE_T` this makes it easier to identify them later. It also looks like we really need to typedefs here. *Yeah legacy formats are great* -.-